cabot/cabot_config.py

```diff
@@ -5,6 +5,8 @@
 GRAPHITE_PASS = os.environ.get('GRAPHITE_PASS')
 GRAPHITE_FROM = os.getenv('GRAPHITE_FROM', '-10minute')
 JENKINS_API = os.environ.get('JENKINS_API')
+if not JENKINS_API.endswith('/'):
```

Comment:
I put it here because 2 separate places in the code require a slash at the end:

```
./cabot/cabotapp/jenkins.py:    endpoint = settings.JENKINS_API + 'job/%s/api/json' % jobname
./cabot/cabotapp/templatetags/extra.py:    return '{}job/{}/'.format(settings.JENKINS_API, jobname
```
but if you think in the code would be better I can add a slash in both places